anaUltraScurve.py

```diff
@@ -249,9 +321,8 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
```

Comment:
removing this line changes how we decide when to mask a channel. Is this replaced somewhere?